Week-12/react-week-12/src/App.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
```

Comment:
Its a good practice to split the file into smaller files which contain smaller components. This way the code is readable and also maintainable.